service/javadsl/api/src/main/java/com/lightbend/lagom/javadsl/api/transport/TransportException.java

```diff
@@ -31,7 +31,7 @@ protected TransportException(TransportErrorCode errorCode, String message) {
     protected TransportException(TransportErrorCode errorCode, Throwable cause) {
         super(cause.getMessage(), cause);
         this.errorCode = errorCode;
-        this.exceptionMessage = new ExceptionMessage(this.getClass().getSimpleName(), cause.getMessage());
+        this.exceptionMessage = new ExceptionMessage(this.getClass().getSimpleName(), errorCode.description());
```

Comment:
Is this `errorCode` the same as the `statusCode`on the HTTP response?

Should we put the same value in two different places?

Reply:
I'm sorry but I don't understand your remark. 
What `statusCode` are you talking about ?
Where do you see "the same value in two different places" ?

And thanks for your careful reviews @ignasi35 and @TimMoore :)